README.md

```diff
@@ -85,7 +85,7 @@ Steps:
         mvn install
 
 
-[Multi-Deposit Instructions]: multi-deposit-instructions.html
+[Multi-Deposit Instructions]: multi-deposit-instructions.md
```

Comment:
Will do, please have a final look at this one: https://github.com/rvanheest/easy-split-multi-deposit/pull/1/files#r53905378